packages/gatsby-plugin-page-creator/src/__tests__/gatsby-node.js

```diff
@@ -10,28 +10,26 @@ describe(`JavaScript page creator`, () => {
       { path: `somedir/dir2/test1.js` },
     ]
 
-    expect(validFiles.filter(file => validatePath(file.path)).length).toEqual(validFiles.length)
+    expect(validFiles.filter(file => validatePath(file.path))).toEqual(
```

Comment:
Also removed all the .length checks. It's a lot safer to validate the actual array shape rather than the length of the array, even though there weren't any changes required here!

e.g. if one of the validFiles was failing _and_ one of the invalid files was passing, we could still get the expected length